Topics/System_Design/Mobile_App_System_Design.md

```diff
@@ -0,0 +1,33 @@
+# Essential Elements of Mobile App System Design
+
+## Introduction
+For the last 15 years, mobile apps have emerged as vital tools in our daily lives. The best mobile apps simplify our lives by helping us with daily tasks. What lies behind these apps is a thoughtful system design that has important considerations that differ from traditional desktop devices and must be taken into account when mobile app system design, namely: UI and Performance.
+
+
+## User Interface Design
+
+### 1. Screen Size
+Compared to desktops, mobile screens are much smaller. The smaller screen size requires specific design considerations, in particular, mobile apps should prioritize essential features and focus on presenting relevant content. For example, in an e-commerce mobile app, the focus should be on the products being sold, in particular, the images of the products with more detailed information being hidden but still seamlessly accessible.
```

Comment:
Adding an illustration here of a 'Good' User Interface Design on Mobile may be a good idea. Since you are talking about UI, I think an image would fit nicely and make this page more readable instead of just having text!

Reply:
I agree. A picture always helps to explain an idea more clearly.

Reply:
Yes I shoud've done that.